content/en/docs/reference/kubectl/resource_printing/fields.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Fields"
+linkTitle: "Fields"
+weight: 3
+type: docs
+description: >
+    Prints based on Field values
+---
+
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Format and print specific fields from Resources
+- Use when scripting with Get
+{{< /alert >}}
+
+# Print Resource Fields
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```

Reply:
Done!